qutebrowser/utils/version.py

```diff
@@ -389,6 +389,11 @@ def _autoconfig_loaded() -> str:
     return "yes" if config.instance.yaml_loaded else "no"
 
 
+def _config_py_loaded() -> str:
+    return "{} has been loaded".format(standarddir.config_py()) if \
+        config.instance.config_py_loaded else "no config.py was loaded"
```

Comment:
This is quite hard to read (I first thought it'd produce "no config.py was loaded has been loaded" before rereading it). I'd prefer:

```python
if config.instance.config_py_loaded:
    return "{} has been loaded".format(standarddir.config_py()) 
else:
    return "no config.py was loaded"
```

Reply:
Now that I see it again, it is indeed hard to read, my bad